base_location/views/better_zip_view.xml

```diff
@@ -7,11 +7,23 @@
         <field name="arch" type="xml">
             <form string="City">
                 <group col="4">
-                <field name="name"/>
-                <field name="code"/>
-                <field name="city"/>
-                <field name="state_id"/>
-                <field name="country_id"/>
+                    <group>
+                        <field name="name"/>
+                        <field name="city_id"
+                               attrs="{
```

Comment:
Don't split this attribute through several lines or the visualization inside UI will be ugly